.github/workflows/CI.yml

```diff
@@ -22,14 +22,14 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: opensearch-project/OpenSearch
-          ref: feature/extensions
+          ref: main
           path: opensearch
 
       - name: Publish to Maven Local
         run: |
           ./opensearch/gradlew --project-dir ./opensearch/ publishToMavenLocal
 
-      - name: Assemble OpenSearch feature/extensions
+      - name: Assemble OpenSearch main
         run: |
           ./opensearch/gradlew --project-dir ./opensearch/ assemble
```

Comment:
Why checkout OpenSearch and build?
Snapshots are already available on maven.

Reply:
Similar comment for SDK, snapshots are now available.

Reply:
Me and @dbwiddis planned to create a new issue and will cater the change in there. I will create the new issue today. Thanks:)